SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,8 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||tracking*.adweb.co.kr^
```

Comment:
Just change `||tracking.adweb.co.kr^$third-party`
And please do not forget `$third-party` if it used on another sites.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/76128/commits/4bd74e3760dfa08dcbc6f5d3f95d09a33bdd8d3e